dev/CommonStyles/Common_themeresources_any.xaml

```diff
@@ -85,7 +85,7 @@
             <Color x:Key="LayerFillColorAlt">#0DFFFFFF</Color>
             <Color x:Key="LayerOnAcrylicFillColorDefault">#09FFFFFF</Color>
             <Color x:Key="LayerOnAccentAcrylicFillColorDefault">#09FFFFFF</Color>
-
+  
```

Comment:
Remove the empty lines with spaces.

Reply:
The line here is left intentionally to better organize resources with different name groupings.

Reply:
Fair enough. I ment remove the spaces.